Test/Notion.IntegrationTests/IBlocksClientTests.cs

```diff
@@ -20,6 +21,9 @@ public IBlocksClientTests()
             };
 
             _client = NotionClientFactory.Create(options);
+
+            // Create Table API is not supported by Notion. So, please create a Table and provide the block id for it.
+            _tableBlockId = "b23ea341-95e4-4aa0-a448-e8c572d83a18";
```

Comment:
I do not want to hard code the id's in the test case. I understand you must have seen the pageParentId being hard coded in other test cases. But I would suggest if you can write this test case generic enough that it would run on every machine if you have the NOTION_TOKEN.

Reply:
I agree hardcoding is not good, but for tables there is no way of creating it from the API and without table being created i cant write an integration test.

Reply:
@KoditkarVedant , Can you plz suggest how to test in this scenario?